### PR TITLE
Alphabet

### PR DESCRIPTION
**Pull request by kristofv8**

### CODE REVIEW

**Review by rsieradzki-lysn — COMMENTED** [review]
It's better to use '_' as a separator for filenames, branches, etc.. so 
Convert a bollean to a string.py -> convert_bollean_to_string.py
spaces are problematic, they depend on operational system etc..

File: alpha.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ef position(alphabet):
```
Comment (rsieradzki-lysn) [inline alpha.py:1]: it's unfinished, ef -def ,missing implementation

File: alphabet.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def position(alphabet):
+    slownik_litera_to_pozycja = {
+        'a': 1,
+        'b': 2,
+        'c': 3
```
Comment (rsieradzki-lysn) [inline alphabet.py:5]: unfinished, add missing letters to position mapping d: 4, ...etc and test if that works

**Comment by kristofv8** [comment]
I've changed everything for today. I know it's bad and doesn't go through the tests but i will be working with it also tomorrow. For now has to be like that. I am gonna keep in my mind  this notice about spaces.

**Pull request merged**